client/layout/guided-tours/positioning.js

```diff
@@ -32,25 +32,25 @@ const helpers = {
 	yBelow: ( bottom ) => {
 		return bottom + DIALOG_PADDING;
 	},
-	xAboveBelow: ( left, right ) => {
+	xAboveBelow: ( left, right, width ) => {
 		if ( ( left + DIALOG_WIDTH + DIALOG_PADDING ) < document.documentElement.clientWidth ) {
 			return left + DIALOG_PADDING;
 		} else if ( right - DIALOG_WIDTH - DIALOG_PADDING > 0 ) {
-			return right - DIALOG_WIDTH - DIALOG_PADDING;
+			return right - ( DIALOG_WIDTH - width );
 		}
 		return DIALOG_PADDING;
 	},
 };
 
 const dialogPositioners = {
 	below: ( rect ) => {
-		const x = helpers.xAboveBelow( rect.left, rect.right );
+		const x = helpers.xAboveBelow( rect.left, rect.right, rect.width );
```

Comment:
Does the `width` change here maybe result in different widths for the steps? IIRC we kept them constant so the text and two buttons will always fit in. I now see this: 

<img width="731" alt="screen shot 2016-12-06 at 1 15 42 pm" src="https://cloud.githubusercontent.com/assets/23619/20925615/47063be8-bbb8-11e6-9578-97eef1c54fbf.png">